### PR TITLE
Rename MCS to Console in constants.go

### PR DESCRIPTION
**Pull request by kerneltime**

### CODE REVIEW

File: pkg/apis/minio.min.io/v1/constants.go [hunk 1]
```diff
@@ -112,23 +112,23 @@ const ConsoleTenantLabel = "v1.min.io/console"
 // MCSPort specifies the default Console port number.
```
Comment (cesnietor) [inline pkg/apis/minio.min.io/v1/constants.go:112]: should this also be modified?

Comment (harshavardhana) [reply pkg/apis/minio.min.io/v1/constants.go:112]: let's rename the `mcs` project to console as well :-)

Comment (kerneltime) [reply pkg/apis/minio.min.io/v1/constants.go]: Not resolving mcs project name in this PR :-)
Fixed the port

Comment (harshavardhana) [reply pkg/apis/minio.min.io/v1/constants.go]: That is done mcs -> console

**Pull request merged**